conda-envs/environment-dev-py37.yml

```diff
@@ -3,7 +3,7 @@ channels:
 - conda-forge
 - defaults
 dependencies:
-- aesara>=2.1.0
+- aesara==2.1.0
```

Comment:
I think with https://github.com/pymc-devs/pymc3/pull/4869 we don't need this anymore.